pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, writeText, fetchurl,
+  libpng, fftw,
+  mpiSupport ? false, mpi ? null
+}:
+
+assert mpiSupport -> mpi != null;
+
+stdenv.mkDerivation rec {
+  version = "16Feb16";
```

Comment:
`version` should satisfy `old < new`, will that be the case here? Perhaps use ISO 8601 instead.